docs/src/index.md

```diff
@@ -396,11 +397,37 @@ julia> @rtransform df @passmissing x = parse(Int, :x_str)
    3 │ missing  missing
 ```
 
+## Creating multiple columns at once with `@astable`
+
+Often new variables may depend on the same intermediate calculations. `@astable` makes it easy to create multiple
+new variables in the same operation, yet have them share
+information. 
+
+In a single block, all assignments of the form `:y = f(:x)` 
+or `$y = f(:x)` at the top-level are generate new columns.
```

Comment:
```suggestion
or `$y = f(:x)` at the top-level generate new columns.
```

Reply:
maybe add what `$y` has to resolve to (I understand it has to be `Symbol`, or strings are also accepted?)